### ISSUE TITLE
Publish some actual portfolio items that don't suck balls

### ISSUE THREAD

**Issue by brashrebel** [realbig/RBM-Theme#36]
If this remains on my to-do list it may never get done. Our portfolio online absolutely sucks so much it's a joke. It is all clients from 2012 and 2013. I want to replace it all with clients from 2015 and 2016. That means someone needs to:

1. Pick clients that we did even reasonably decent work for.
2. Get their permission.
3. Write something about what we did.
4. Include a nice main image for the portfolio archive and some screenshots for the portfolio single.
5. Publish on our site.
6. Purge old portfolio items that are truly garbage.

Can you do this????

**Comment by joelworsham** [comment]
Much angst. Such wow.

**Comment by d4mation** [comment]
![image](https://cloud.githubusercontent.com/assets/7770631/21656618/f102b0b4-d28c-11e6-9d42-f8b549686f3a.png)

**Comment by BigActual** [comment]
Closing and combining into #37

**Comment by brashrebel** [comment]
These are not the same. Home page portfolio items are a visual showcase of some sites that look pretty. Our actual portfolio page is a list of our clients with corresponding write ups. They are different. The home page issue just requires a series of our best looking _custom_ work. That means nothing that was designed by a third party or was built with a stock theme. The portfolio page is entirely different.

**Comment by BigActual** [comment]
Yeah, thanks for the domain. Sorry I had to ask, but it's hard to google your way around Evangelical Pat Robertson

**Comment by BigActual** [comment]
Ooops. Your right. Been awhile since I looked at it.
